app/components/procedure/card/sva_svr_component/sva_svr_component.fr.yml

```
@@ -1,6 +1,6 @@
 ---
 fr:
-  title: "Silence Vaut Accord"
+  title: "SVA ou SVR"
```

Proposition plus explicite mais peut être que ça rentre pas

```suggestion
  title: "Silence Vaut Accord ou Rejet"
```

---

ça rentre pas mais c'est plus clair, je prends